src/main/java/org/kainos/ea/api/DeliveryService.java

```diff
@@ -1,2 +1,42 @@
-package org.kainos.ea.api;public class DeliveryService {
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.DeliveryEmployee;
+import org.kainos.ea.cli.DeliveryRequest;
+import org.kainos.ea.client.DeliveryEmployeeDoesNotExistException;
+import org.kainos.ea.client.FailedToCreateDeliveryEmployeeeException;
+import org.kainos.ea.db.DeliveryDao;
+
+import java.sql.SQLException;
+
+public class DeliveryService {
+    private DeliveryDao deliveryDao = new DeliveryDao();
```

Comment:
Can this be improved to follow the D in solid. And inject the DAO through the constructor